python/ray/tune/ray_trial_executor.py

```diff
@@ -101,7 +102,10 @@ def logger_creator(config):
 
         # Logging for trials is handled centrally by TrialRunner, so
         # configure the remote runner to use a noop-logger.
-        return cls.remote(config=trial.config, logger_creator=logger_creator)
+        return cls.remote(
```

Comment:
I'd like this to go in (the ability to notify the Trainable of its resource allocation) although I'm not sure if this belongs in this PR. 

One thing that this PR currently assumes is that the TorchRunner will automatically create N workers depending on its resource allocation.

cc @ericl